src/app/i18n/provider/TranslationProvider.tsx

```diff
@@ -14,7 +14,8 @@ interface TranslationProviderProps {
 export const TranslationProvider: React.FC<TranslationProviderProps> = ({ children }) => {
   const { t } = useTranslation();
 
-  const translateList = (key: string) => t(key, { returnObjects: true }) as string[];
+  const translateList = (key: string, props?: Record<string, unknown>) =>
+    t(key, { returnObjects: true, replace: props }) as string[];
```

Comment:
It is used to replace the elements that are put in square brackets. Since we are modifying the t object, we have to specify which options we want to use (the t object does it by itself), but in this case we must specify it.